libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -189,10 +190,20 @@ public OAuthWebviewHelper(Activity activity, OAuthWebviewHelperEvents callback,
         }
     }
 
+    public OAuthWebviewHelper(Context context, OAuthWebviewHelperEvents callback, LoginOptions options) {
```

Comment:
This class has the code that runs at the end of a auth flow (creating the user etc). That could happen in the SP app after getting the code from the IDP without having a webview.
=> we will need to refactor that code further e.g. the name of the class "OAuthWebviewHelper" is a bit misleading now.